File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.java
import java.util.Arrays;

class Solution {
    public int solution(String[] strArr) {
        int[] answer = new int[31];
        for(int i = 0; i < strArr.length; i++){
            answer[strArr[i].length()]++;
        }
        int max = Arrays.stream(answer).max().getAsInt();
        return max;
    }
}

File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<string> strArr) {
    vector<int> answer;
    answer.resize(31);
    for(int i = 0; i < strArr.size(); i++){
        answer[strArr[i].length()]++;
    }
    auto maxIterator = max_element(answer.begin(), answer.end());
    int maxValue = *maxIterator;
    return maxValue;
}